### PR TITLE
Add Operate First Data Science Community to website

### LINKED ISSUE #318: Create landing page for public DS Meet Up [EPIC]
We would like to start hosting regular Data Science Meet Ups as part of the operate first community. These meet ups should include quick highlights on the state of ongoing projects and new features available to OPF DS users as well as deep dives into a particular project or topic each session. All sessions should be public and recorded.    

- [x] Write landing page markdown for a public opf ds meet up https://github.com/aicoe-aiops/cloud-first-data-science-community/pull/3 
- [x] Add landing page to site as operate-first.cloud/data-science/community

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Resolves #318

### CODE REVIEW

File: config/data-science.yaml [hunk 1]
```diff
@@ -104,3 +104,9 @@
       href: /data-science/data-skipping/site/docs/getting-started/operatefirst-getting-started.md
     - label: Get Started
       href: /data-science/data-skipping/notebooks/python/Xskipper - Python Sample.ipynb
+- label: Operate First Data Science Community
```
Comment (MichaelClifford) [inline config/data-science.yaml]: Should we change the label to "Get Involved"? WDYT?

File: config/data-science.yaml [hunk 2]
```diff
@@ -104,3 +104,9 @@
...
+  links:
+    - label: Data Science Community Landing Page
```
Comment (MichaelClifford) [inline config/data-science.yaml]: Let's change this to just say "Meetup". Landing Page isn't really an external facing word.

File: config/data-science.yaml [hunk 3]
```diff
@@ -104,3 +104,9 @@
...
+      href: /data-science/operate-first-data-science-community/docs/meetup-landing-page.md
+    - label: Content
```
Comment (MichaelClifford) [inline config/data-science.yaml]: Let's change this label to say "agenda"

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: config/data-science.yaml (hunk 2 above)
Comment (aakankshaduggal) [reply config/data-science.yaml]: Sure

File: config/data-science.yaml (hunk 1 above)
Comment (aakankshaduggal) [reply config/data-science.yaml]: Sounds good :+1:

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**